internal/config/config.go

```diff
@@ -716,18 +733,21 @@ func validateBGPAdvPerPool(adv *BGPAdvertisement, pool *Pool) error {
 	return nil
 }
 
+var ErrInvalid = errors.New("invalid string")
```

Comment:
nit: no need to make these public, also I'd name them more explicitly:
errInvalidCommunity (or just invalidCommunity / invalidCommunityFormat)

Reply:
done